app/controllers/genre_controller.rb

```diff
@@ -0,0 +1,41 @@
+class GenreController < ApplicationController
+  def index
+    @genre = Genre.order(id: :desc)
+
+    render json: @genre
+end
+
+# GET /cinema_halls/1
+def show
+end
+
+# POST /cinema_halls
+def create
+    render json: Genre.new(genre_params)
+end
+
+# PATCH/PUT /cinema_halls/1
+def update
+  if @genre.update(genre_params)
+    render json: @genre
+  else
+    render json: @genre.errors, status: :unprocessable_entity
+  end
+end
+
+# DELETE /cinema_halls/1
+def destroy
+  @genre.destroy
+end
+
+private
+  # Use callbacks to share common setup or constraints between actions.
+  def set_genre
```

Comment:
Looks like you forgot to use this method in before_action callback or anywhere tbh :D

Reply:
I wasnt focusing on genre yet. I tried to make ticket_controller work, it's almost done. I need to focus on small things.

Reply:
Sure thing, but it is probably better to not commit the changes that are not finished yet 😃 or maybe add some comments like "todo" or "wip"?

Reply:
Rules of pushing commits transcends time and space :D